module/category/src/Infrastructure/Persistence/Query/DbalTreeQuery.php

```diff
@@ -70,6 +70,35 @@ public function findTreeIdByCode(string $code): ?CategoryTreeId
         return null;
     }
 
+    public function autocomplete(
+        Language $language,
+        string $search = null,
+        int $limit = null,
+        string $field = null,
+        ?string $order = 'ASC'
+    ): array {
+        $query = $this->connection->createQueryBuilder()
+            ->select('id, code, COALESCE(name->>:language, null) as label')
+            ->from(self::TREE_TABLE, 'c')
+            ->setParameter(':language', $language->getCode());
+
+        if ($search) {
+            $query->Where('code ILIKE :search');
```

Comment:
should be lower case. I though it is cover in one of the checks but apparently wrong